library/cwm/examples/object_api.rb

```diff
@@ -17,7 +17,7 @@ def initialize
   end
 
   def label
-    _("Lucky number")
+    "Lucky number"
```

Comment:
Maybe should at least add a FIXME comment saying that this text should be marked for translation when used in production code. Developers usually do just copy&paste :wink:, this could avoid some mistakes in that case.